File: Benchmarks/enterprise/python/django/testcode/benchmark_test_10454.py
# SPDX-License-Identifier: Apache-2.0
from django.http import JsonResponse
import os


def BenchmarkTest10454(request):
    user_id = request.GET.get('id', '')
    if user_id:
        data = user_id
    else:
        data = ''
    os.remove(str(data))
    return JsonResponse({"saved": True})

File: Benchmarks/enterprise/java/spring/testcode/BenchmarkTest10454.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest10454 {

    @GetMapping("/BenchmarkTest10454")
    public void BenchmarkTest10454(@RequestHeader("Host") String host, HttpServletRequest request, HttpServletResponse response) throws Exception {
        response.sendError(403, "directory listing forbidden");
    }
}
